src/common/fs_manager.cpp

```diff
@@ -297,17 +321,29 @@ bool fs_manager::for_each_dir_node(const std::function<bool(const dir_node &)> &
 
 void fs_manager::update_disk_stat(bool check_status_changed)
 {
-    reset_disk_stat();
+    _total_capacity_mb = 0;
+    _total_available_mb = 0;
```

Comment:
Is there any other place except this function use it? If not, how about define it as a local variable?

Reply:
Yes, both are used in [replica_stub.cpp#L1156](https://github.com/apache/incubator-pegasus/blob/master/src/replica/replica_stub.cpp#L1156) as the response for `query_disk_info_rpc`.

Reply:
OK